README.md

```diff
@@ -1,4 +1,4 @@
-Zcash 1.0.10
+Zcash 1.0.10-1
 =============
```

Comment:
This is syntactically incorrect Markdown because the line of = doesn't reach to the end of the title.